docker-compose.yml.example

```diff
@@ -0,0 +1,25 @@
+version: '2.1'
+services:
+  app:
+    build:
+      context: .
+      args:
+        BUILD_ENV: development
```

Comment:
I think it would be helpful to add comments with recommended production settings, such as
* setting `BUILD_ENV` to `production`
* not volumizing the codebase
* not mapping port 3000
* maybe volumizing uploaded files?

but that could come in a future PR once all that is worked out.

Reply:
mostly I don't know what that means 😅 so maybe that should come in a future PR.